src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -1541,6 +1541,13 @@ tr.status-warning td {
 
 {# Toggles
    ========================================================================= #}
+.sf-toggle-button {
```

Comment:
We already have `btn-link` implementing that behavior of buttons looking like a link (and with the proper hover state)

Reply:
I have changed and force pushed again - thanks.